lib/world.rb

```diff
@@ -141,6 +141,7 @@ def #{snake_case}(*args, &block)
     alias scc security_context_constraints
     alias kubeapiserver kube_a_p_i_server
     alias openshiftapiserver open_shift_a_p_i_server
+    alias consolenotifications console_notification_console_openshift_io
```

Comment:
`console_notifications`

Reply:
Actually `console_notification`. This alias here is used when we work with such objects in scenarios. e.g.
```
Then the expression should be true> console_notification.whatever == something
```
That's why I suggest to put it singular form here.

Reply:
I see `openshiftapiserver` and `kubeapiserver` do not follow this convention. I need to go back and change them. Shouldn't be hard to grep them out.